src/ui/components/select-dropdown/index.jsx

```diff
@@ -289,7 +294,14 @@ class SelectDropdown extends Component {
 		);
 	}
 
-	toggleDropdown() {
+	onContainerClick( event ) {
```

Comment:
Should we rename this to just `onClick`? The click's target can be the own SelectDropdown component, its parent, etc.